prusti-tests/tests/cargo_verify/veribetrfs/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "veribetrfs"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+
+[workspace]
```

Comment:
Could you make sure that all files end with a newline? If you open the [files](https://github.com/viperproject/prusti-dev/pull/923/files) tab of the PR GitHub shows a red sign where the newline is missing.